src/seed/time-sheets.js

```diff
@@ -0,0 +1,62 @@
+import mongoose from 'mongoose';
+
+export default [{
+  _id: mongoose.Types.ObjectId('635321207ca59575d18db35b'),
+  description: 'Example for testing with jest in timesheet',
+  date: '2022-05-15',
+  hours: 18,
+  task: {
```

Comment:
We don't need to populate the entities. We should have something like:
```suggestion
  task: mongoose.Types.ObjectId('635321f4e4e54b3f10b47f9e')
```
The same for the other entities.

Reply:
Perfect, I'm going to modify it